### PR TITLE
Clean up after fork (NR-303974)

### PR DESCRIPTION
**Pull request by kilokang**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Global owners
+* @newrelic/coreint
```
Comment (sigilioso) [inline CODEOWNERS]:
```suggestion
* @newrelic-forks/coreint
```

File: .github/workflows/release.yaml
Comment (sigilioso) [inline .github/workflows/release.yaml:1]: Should we also upload the artifacts to the release itself?

Comment (kilokang) [reply .github/workflows/release.yaml:1]: `goreleaser` have everything in its pipeline enabled by default, you have to disable explicitly to that step to not happen. Like we have in [newrelic-prometheus-exporters-packages](https://github.com/newrelic/newrelic-prometheus-exporters-packages/blob/7da0106ebcc2f87697d0e17d503adfd918d53ee1/scripts/pkg/.goreleaser.yml.template#L95-L96).

`release` step should be enabled with the configuration that this branch provides :)

File: CODEOWNERS (hunk 1 above)
Comment (kilokang) [reply CODEOWNERS]: I had to add the team as admin to the repo for this file to be valid too.

Good catch!

**Review by sigilioso — APPROVED** [review]
LGTM

**Pull request merged**